wordpress-mobile/WordPress-Android
# Log In What's my site address? dialog image not showing properly

Fixes #12422

Find your site address dialog was not displaying image with correct dimension on some devices with some android versions.  For e.g. Oreo on Motorola.
Updated vector drawable width and height to fix the issue.

To test:
Welcome Screen -> Enter your existing site address -> Find your site address (on Log In) -> **Verify dialog image**
(This needs to be tested particularly on android Oreo ver 8.1, api 26, 27 besides other versions)
<table><tr><td><img width="320" src="https://user-images.githubusercontent.com/990349/118206310-90fc7600-b4a5-11eb-91f6-d9711834682e.png"></td>
<td><img width="320" src="https://user-images.githubusercontent.com/990349/118206327-98bc1a80-b4a5-11eb-9ee1-7645563eadb1.png"></td>
<td><img width="320" src="https://user-images.githubusercontent.com/990349/118206333-9c4fa180-b4a5-11eb-9680-01fe133411cc.png"></td></tr></table>

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
NA

3. What automated tests I added (or what prevented me from doing so)
NA

PR submission checklist:
